lib/downloader/scraper.rb

```diff
@@ -32,7 +32,7 @@ def sign_in(signin_url)
       form = page.form
       form.field_with('username').value = @username
       form.field_with('password').value = @password
-      form.checkbox_with('remember_me').check
+      (checkbox = form.checkbox_with('remember_me')) && checkbox.check
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.